lisp/ess-custom.el

```diff
@@ -2108,27 +2113,26 @@ for help files.  The default value is nil for other systems."
 
 ;;;;; names for communication using MS-Windows 9x/NT ddeclient mechanism
 
-(defcustom inferior-ess-ddeclient nil
+(defcustom inferior-ess-ddeclient "Initial"
```

Comment:
> We just need to check I didn't break anything.

Aha great. @rmheiberger could you please make sure that you test/work on this branch or wait till it's merged? Simple reversion to 2010 will surely not work anymore.